portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,31 +17,42 @@
 import com.google.gson.Gson;
 import java.io.IOException;
 import java.util.List; 
-import java.util.Arrays; 
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. */
-@WebServlet("/data")
+@WebServlet("/comment-data")
 public class DataServlet extends HttpServlet {
-  private static final List<String> ANSWERS = 
-      Arrays.asList("A stick!", "A wet log!", "(some) Frogs!");                                                        
+  private static List<String> comments = new ArrayList<>();                                                     
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Convert the array of answers to JSON
-    String jsonAnswers = convertToJson(ANSWERS);
+    // Convert the List of comments to JSON.
+    String jsonComments = convertToJson(comments);
 
-    // Send the JSON as the response
+    // Convert list to JSON and send to comment-data page.
     response.setContentType("application/json;");
-    response.getWriter().println(jsonAnswers);
+    response.getWriter().println(jsonComments);
   }
 
-  private String convertToJson(List<String> answers) {
+  private String convertToJson(List<String> comments) {
     Gson gson = new Gson();
-    String json = gson.toJson(answers);
+    String json = gson.toJson(comments);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the html text form.
+    String newComment = request.getParameter("new-comment");
```

Comment:
wanna do any sanity check for the comment? what if they left just a space?

Reply:
I was thinking of having some sort of verifyInput() function once the MVP is completed. Do you think it makes sense for me to put this feature in now or would it be okay to include this in a latter PR after step 6 is complete?

Reply:
sg.

Reply:
Created issue #15 .